src/Sample.Presentation/CoreStartup.cs

```diff
@@ -116,6 +117,7 @@ private static void InitializeLoggerFactories(IServiceProvider services)
 		BackButtonManagerConfiguration.LoggerFactory = loggerFactory;
 		DynamicMvvmConfiguration.LoggerFactory = loggerFactory;
 		DataLoaderConfiguration.LoggerFactory = loggerFactory;
+		DynamicMapConfiguration.LoggerFactory = loggerFactory;
```

Comment:
Shouldn't you add the same thing for MapService (and perhaps StaticMap)?

Reply:
Forgot it, StaticMap doesn't use a ILogger.